paddle/fluid/platform/mkldnn_reuse.h

```diff
@@ -1332,15 +1334,18 @@ class ConvMKLDNNTemplateHandler : public MKLDNNHandler {
       int mask = output_shift_scale.size() > 1 ? 1 << 1 : 0;
       conv_attr.set_output_scales(mask, output_shift_scale);
     }
-    // Fusion with Elementwise layer relies on adding a sum post-operation with
-    // the scale parameter. It is assumed that when fuse_residual_connection is
+    // Fusion with Elementwise layer relies on adding a sum post-operation
```

Comment:
Please, refactor these commented lines so there won't be any line containing only single word